garak/attempt.py

```diff
@@ -129,7 +129,7 @@ def last_message(self, role=None) -> Message:
             raise ValueError("No messages available")
         if not role:
             return self.turns[-1].content
-        for idx in range(len(self.turns), 0):
+        for idx in range(len(self.turns) - 1, -1, -1):
```

Comment:
space before the first one makes all the difference for legibility. nice. love this func now